src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { nanoid } from 'nanoid';
+import heroImage from '../../images/hero/coctail/heroImage.png';
+import {
+  Alphabet,
+  AlphabetElement,
+  Container,
+  NumberElement,
+  Numbers,
+  Text,
+  Title,
+} from './Hero.styled';
+
+const englishAlphabet = [
+  'A',
+  'B',
+  'C',
+  'D',
+  'E',
+  'F',
+  'G',
+  'H',
+  'I',
+  'J',
+  'K',
+  'L',
+  'M',
+  'N',
+  'O',
+  'P',
+  'Q',
+  'R',
+  'S',
+  'T',
+  'U',
+  'V',
+  'W',
+  'X',
+  'Y',
+  'Z',
+];
+
+const numbers = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0'];
+
+export default function Hero() {
+  return (
+    <div>
+      <Container>
+        <div>
+          <Title>A party without cocktails is not like a party</Title>
+          <Text>Search your favorite cocktail by ABC </Text>
+          <Alphabet>
+            {englishAlphabet.map(letter => {
+              return <AlphabetElement key={nanoid()}>{letter}</AlphabetElement>;
```

Comment:
There is no need to use `nanoid()` since every letter in this array is already unique. You can use the `letter` itself.